planet-notebook-docker/Dockerfile

```diff
@@ -18,6 +18,11 @@ USER $NB_UID
 RUN conda config --set channel_priority strict && \
     conda install -y -c conda-forge --file /tmp/requirements.txt
 
+# install the following libraries w/ pip to support analytics/user-guide nb's
+# build fails when these dependencies are added to requirements/ installed with conda
+# python -m pip as per https://github.com/pypa/pip/issues/5599
+RUN python -m pip install descartes holoviews geoviews xarray datashader
```

Comment:
I'm curious how the build fails. Is one of the packages not available or is it unable to resolve the dependencies? Also, if any of these packages don't cause a build failure, could you put them in conda? We are trying to keep all the libraries installed in conda to minimize conflicts as much as possible.

Reply:
Upon revisiting this, I was able to slim down these dependencies and add them to requirements.txt. By pinning the requirements, the conda build does not get `killed`.